### PR TITLE
Add ecto_ch to adapters list

### PR DESCRIPTION
**Pull request by ruslandoga**
👋

This PR adds https://github.com/plausible/ecto_ch to the list of available adapters. There have been few changes to both ecto_ch and the underlying driver, Ch, so I think it  should be stable enough at this point :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,13 +85,14 @@ See the [getting started guide](https://hexdocs.pm/ecto/getting-started.html) an
...
-| SQLite3    | Ecto.Adapters.SQLite3  | [ecto_sql][ecto_sql] (requires Ecto v3.5+) + [ecto_sqlite3][ecto_sqlite3] |
-| ETS        | Etso                   | [ecto][ecto] + [etso][etso]                                               |
+| Database   | Ecto Adapter             | Dependencies                                                              |
+| :--------- | :----------------------- | :------------------------------------------------------------------------ |
+| PostgreSQL | Ecto.Adapters.Postgres   | [ecto_sql][ecto_sql] (requires Ecto v3.0+) + [postgrex][postgrex]         |
+| MySQL      | Ecto.Adapters.MyXQL      | [ecto_sql][ecto_sql] (requires Ecto v3.3+) + [myxql][myxql]               |
+| MSSQL      | Ecto.Adapters.Tds        | [ecto_sql][ecto_sql] (requires Ecto v3.4+) + [tds][tds]                   |
+| SQLite3    | Ecto.Adapters.SQLite3    | [ecto_sql][ecto_sql] (requires Ecto v3.5+) + [ecto_sqlite3][ecto_sqlite3] |
+| ETS        | Etso                     | [ecto][ecto] + [etso][etso]                                               |
+| ClickHouse | Ecto.Adapters.ClickHouse | [ecto_sql][ecto_sql] (requires Ecto v3.9+) + [ch][ch]                     |
```
Comment (josevalim) [inline README.md:95]:
```suggestion
| ClickHouse | Ecto.Adapters.ClickHouse | [ecto_ch][ecto_ch]                                                     |
```

File: README.md [hunk 2]
```diff
@@ -100,6 +101,7 @@ You need to add both Ecto and the database adapter as a dependency to your `mix.
 [tds]: https://github.com/livehelpnow/tds
 [ecto_sqlite3]: https://github.com/elixir-sqlite/ecto_sqlite3
 [etso]: https://github.com/evadne/etso
+[ch]: https://github.com/plausible/ch
```
Comment (josevalim) [inline README.md:104]:
```suggestion
[ecto_ch]: https://github.com/plausible/ecto_ch
```

File: README.md (hunk 1 above)
Comment (josevalim) [reply README.md:95]: I assume we only need to add ecto_ch as a dependency. :)

File: README.md (hunk 2 above)
Comment (ruslandoga) [reply README.md:104]: 🙈

File: README.md (hunk 1 above)
Comment (ruslandoga) [reply README.md]: Right 🙈

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**